test/test_multirate.py

```diff
@@ -522,6 +523,7 @@ def rhs_slow(t, fast, slow):
 @pytest.mark.parametrize("static_dt", [True, False])
 def test_implicit_accuracy(order, hist_length, system, static_dt, step_ratio,
                            plotting=False):
+    pytest.importorskip("scipy")
```

Comment:
As above: Not a fan. Just add scipy [here](https://github.com/inducer/leap/blob/f1b5a3efc3ef2b50e2fc489523fadcab3ced07d1/.github/workflows/ci.yml#L56).

Reply:
Added!

Reply:
(and the implicit tests are now not being skipped/are passing)